src/components/scopedVariables/constants.tsx

```diff
@@ -55,3 +55,31 @@ export const EMPTY_FILE_STATUS = {
     },
     status: false,
 }
+
+export const VARIABLES_TEMPLATE = `Variables:
```

Comment:
Renamed it more previsly

Reply:
Renamed it to SCOPED_VARIABLES_TEMPLATE_DATA in upcoming PR since it has been used in other places as well